script/execute.go

```diff
@@ -135,6 +107,11 @@ func Execute(pkScript []byte, witness [][]byte, interactive bool) error {
 		}
 
 		output.DrawTable(table, clearLines)
+		if currentStep > 1 {
```

Comment:
Maybe this should only be printed when `interactive == true`?

Reply:
Definitely, good catch.